src/include/txn.h

```diff
@@ -240,6 +240,13 @@ struct __wt_txn_op {
 			} mode;
 		} truncate_row;
 	} u;
+
+#ifdef HAVE_DIAGNOSTIC
+/* AUTOMATIC FLAG VALUE GENERATION START */
+#define	WT_TXN_OP_REPEATED	0x1u
```

Comment:
I prefer WT_TXN_OP_KEY_REPEATED, to indicate the key is repeated, not the operation.